docker/cortx-deploy/cortx-componenet-rpms.txt

```diff
@@ -5,3 +5,4 @@ cortx-s3server
 cortx-csm_agent
 cortx-provisioner
 cortx-ha
+cortx-py-utils-test
```

Comment:
This change will add `cortx-py-utils-test` package in production cortx-all image as well. We need to add it only for cortx-py-utils pre-merge job.